src/NetSparkle/Enums/MatchingResult.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NetSparkleUpdater.Interfaces;
+
+namespace NetSparkleUpdater.Enums
+{
+    /// <summary>
+    /// Provides the return values for the GetAvailableUpdates call on the IAppCastHandler.  When an appcast is downloaded,
+    /// the IAppCastHandler will work out which AppCastItem instances match criteria for an update.  
+    /// </summary>
+    /// <seealso cref="IAppCastHandler.GetAvailableUpdates"/>
+    public enum MatchingResult
```

Comment:
What do you think about using the name `CheckValidUpdateResult` instead? 🤔 I'm not too keen on the current name, but not to sure about a better one.

Reply:
Maybe there is a more descriptive term.  I've already read the comment above about valid vs match... hmmm... if I reflect on what is being done; we are filtering an update - the only reasons I could think of at this stage were: we are running on Mac (haha) and want to discard Windows updates (unrealistic I suspect); or moving from beta -> stable channel and we want to discard versions that are too new.

? 

FilterResult
ContextResult

I'm open for ideas too.